### PR TITLE
Add additional config parameters for HomematicIP

### PR DESCRIPTION
**Pull request by SukramJ**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#21076

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/homematicip_cloud.markdown [hunk 1]
```diff
@@ -65,6 +65,8 @@ Add the information to your `configuration.yaml` file:
 homematicip_cloud:
   - accesspoint: IDENTIFIER
     authtoken: AUTHTOKEN
+    enable_group_switches: true
```
Comment (klaasnicolaas) [inline source/_components/homematicip_cloud.markdown:68]: This is an optional variable and should not be included in the first example yaml config.

File: source/_components/homematicip_cloud.markdown [hunk 2]
```diff
@@ -65,6 +65,8 @@ Add the information to your `configuration.yaml` file:
...
+    enable_group_sec_sensors : false
```
Comment (klaasnicolaas) [inline source/_components/homematicip_cloud.markdown:69]: Same

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by worm-ee** [comment]
Thanks for your contribution! 
I would just add the group devices, there is no need to add the config options. For hassio users it might be even confusing - the have only the options to remove and add the platform again.

**Comment by SukramJ** [comment]
I removed the config options, so there is no need for the PR any more

**Pull request closed without merging**